observation_portal/requestgroups/viewsets.py

```diff
@@ -254,6 +254,17 @@ def observations(self, request, pk=None):
             return Response([o.as_dict(no_request=True) for o in observations if o.state != 'CANCELED'])
         return Response([o.as_dict(no_request=True) for o in observations])
 
+    @action(detail=False, methods=['post'])
+    def mosaic(self, request):
+        # Check that the mosaic parameters specified are valid
+        mosaic_serializer = import_string(settings.SERIALIZERS['requestgroups']['Mosaic'])(data=request.data)
+        if not mosaic_serializer.is_valid():
+            return Response(mosaic_serializer.errors, status=400)
```

Comment:
Can use `status=status.HTTP_400_BAD_REQUEST` here, where the import is `from rest_framework import status`

Reply:
okay, went ahead and switched all 400's to that within that file.